client/my-sites/current-site/index.jsx

```diff
@@ -104,8 +104,9 @@ module.exports = React.createClass( {
 		);
 	},
 
-	trackHomepageClick: function() {
+	previewSite: function( event ) {
 		analytics.ga.recordEvent( 'Sidebar', 'Clicked View Site' );
```

Comment:
Oh, right, it's an actual *event* name—I'd assumed it was just a string representation for easier reading. Yes, in that light it makes total sense to keep!